ansible_runner/config/_base.py

```diff
@@ -150,16 +150,24 @@ def _prepare_env(self, runner_mode='pexpect'):
         if self.runner_mode == 'pexpect':
             try:
                 if self.passwords and isinstance(self.passwords, dict):
-                    self.passwords = self.passwords.update(self.loader.load_file('env/passwords', Mapping))
+                    # Load password file and then the passwords dict argument for precedence
```

Comment:
What's the reason for the change in precedence?

Reply:
Because it's useless to provide password as dictionnary if a file is already present if we don't not modify precedence. 
Argument provided by "user" in python code should always be more important as config file.

Reply:
@Shrews made a good point to me

```python
self.settings.update(self.loader.load_file('env/settings', Mapping))
```

differs from your precedence rules for passwords here.

I agree with you sentiment

> Argument provided by "user" in python code should always be more important as config file.

But that would apply to settings, just the same that it applies to passwords.

Reply:
@AlanCoding you're right.
I didn't change settings because I don't not use it and the initial fix was not intend to change settings. I just changed the obvious bug about settings.
If you insist I change change precedence with settings to, but I just need to know how to test it.

Reply:
IMO, all entries in `env` should have the same precedence. I feel like this PR should just fix the current bug. Changing precedence for file vs. args should likely be a separate PR and make sure that precedence is treated the same way across all possible input files.